Gemfile

```diff
@@ -4,6 +4,7 @@ source "https://rubygems.org"
 
 gem 'lotus-model', github: "lotus/model"
 gem "lotusrb", github: "lotus/lotus"
+gem 'lotus-controller', '0.2.0'

```

Comment:
This was the important one. Without adding this, heroku installed not version 0.2, but the latest one, which is not the version we want :wink: